app/controllers/labels_controller.rb

```diff
@@ -1,4 +1,41 @@
 class LabelsController < ApplicationController
+  before_action :get_label, only: [:destroy]
```

Comment:
since it's only applicable for the `destroy` action,
should we move it into the action itself? e.g:
```ruby
def destroy
    @label = Label.find_by(id: params[:id])

    if @label.blank?
      redirect_to root_path, notice: "Label #{params[:id]} was not found."
    end
    
    if @label.destroy
        redirect_to root_path, notice: "Label `#{@label.title}` was successfully deleted."
    end
end
```